src/components/pages/schedule/list.vue

```diff
@@ -81,8 +81,8 @@ export default {
     },    
     format (unixTimestamp) {
       if (unixTimestamp) {
-        var utcTime = moment(unixTimestamp).utc()
-        return utcTime.format('DD-MMM-YYYY') + ' ' + utcTime.format('HH:mm')
+          var localTime = moment();
```

Comment:
This will display current time. Not the time of execution

Reply:
That's correct. 🤦

I'll edit the PR.

Reply:
Hold on a bit. I need to think over if we really need to change all timestamps everywhere on r2cloud-ui.. We have multiple places on the web interface where timestamps displayed. And if we change them in one place, then we should change it everywhere. + we need to put label in what timezone we're working. Someone else might come and ask to render all times in UTC :)

Reply:
That's fair. I thought local time would be more appropriate because then I'd know when to expect the satellite. 

I have not examined if the Schedule page is server rendered, but if it is, then it might be difficult to compute local time for non-local users. It might be worthwhile to create a TimeStamp component, and mark it no-ssr.

Reply:
It is rendered purely on client side, so no issues here. We can make it more user-friendly if we 

1. Add label "UTC" + some icon to the header.
2. When user clicks on them, timezone label switches to "local" or "Europe/City" with the changes to the schedule.

What do you think?

Reply:
Is the icon in the table header? I think that would be perfect!

![Screenshot 2020-07-14 at 11 44 43 PM](https://user-images.githubusercontent.com/16903044/87461585-46fbf700-c62c-11ea-862b-d0ee9647a151.png)

Sorry about the late reply, having some network issues at my place.

Reply:
Yes!

Reply:
Awesome! Also, it would be pretty cool if we show the time remaining in a tooltip when we hover over it, or beside it. What do you think?

Reply:
Can you elaborate on "time remaining"? Do you mean the case when observation created, but still processing? If so, then yes - we can add it as a spinner in the "Has data" column.

Reply:
No, I meant the time for the satellite to arrive over head; `scheduled_time - current_time`.

Reply:
Ah. This should be on the different page - "Schedule". This page has only happened observations.

Reply:
Yeah, I meant in schedule. I didn't realise the screenshot was in observations page XD